GLOSSARY.md

```diff
@@ -0,0 +1,86 @@
+## access point
+Networking hardware which creates a local area network over Wifi
```

Comment:
Oh nevermind. I misconstrued the point of this document. 😞 

This is a good definition of an access point. :+1: